python-lib/font_exceptions_dict.py

```diff
@@ -1,27 +1,36 @@
 # -*- coding: utf-8 -*-
-"""Module with constants defining the wordcloud font exceptions for languages not supported by Noto Sans Display"""
+"""Module with constants defining the wordcloud font exceptions for languages not supported by the default font.
+- Default font is NotoSansMerged-Regular-1000upem.ttf, which is a merged font based on various NotoSans fonts:
+    - NotoSansDisplay-Regular
+    - NotoSansArabic-Regular
+    - NotoSansArmenian-Regular
+    - NotoSansBengali-Regular
+    - NotoSansDevanagari-Regular
+    - NotoSansHebrew-Regular
+    - NotoSansSinhala-Regular
+    - NotoSansTamil-Regular
+    - NotoSansThai-Regular
+
+- NotoSansMerged-Regular-2048upem.ttf is a similar font containing fonts that couldn't be merged with precited ones:
```

Comment:
Why is `NotoSansDisplay-Regular` not included in this one? Would this work if there is a mix of Telugu and English?